packages/gatsby-plugin-feed/src/gatsby-ssr.js

```diff
@@ -2,12 +2,12 @@ import React from "react"
 import { defaultOptions } from "./internals"
 
 exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
-  const { feeds } = {
+  const options = {
     ...defaultOptions,
     ...pluginOptions,
   }
 
-  const links = feeds.map(({ output }, i) => {
+  const links = options.feeds.map(({ output }, i) => {
```

Comment:
Just curious: is this a style change only, or was the other implementation harming performance?

Reply:
Ah, just stylistic. I'm going to revert these changes because they're just adding noise to the diff.